libs/common/src/common.constant.ts

```diff
@@ -157,6 +157,11 @@ export enum CommonConstants {
   // EVENTS
   EVENT_AUDIT = 'audit_event',
 
+  // POLYGON KEYWORDS
+  POLYGON = 'polygon',
+  POLYGON_MAINNET = 'mainnet',
```

Comment:
Do we need this?
```
  POLYGON_MAINNET = 'mainnet',
  POLYGON_TESTNET = 'testnet',
```
We already have:
```
  TESTNET = 'testnet',
  STAGINGNET = 'stagingnet',
  BUILDERNET = 'buildernet',
  MAINNET = 'mainnet',
```
[here](https://github.com/credebl/platform/blob/0e03febd6c45f34467ecce866705c544e64a3392/libs/common/src/common.constant.ts#L302)